inc/Theme.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace Bluehost\Maestro;
+
+use Exception;
+use WP_User_Query;
+
+/**
+ * Class for managing Themes
+ */
+class Theme {
+
+	/**
+	 * The id for the theme
+	 *
+	 * @since 1.1.1
+	 *
+	 * @var string
+	 */
+	public $id;
+
+	/**
+	 * The theme name, usually same as the id
+	 *
+	 * @since 1.1.1
+	 *
+	 * @var string
+	 */
+	public $name;
+
+	/**
+	 * The theme's title (or Name in pure WP sense)
+	 *
+	 * @since 1.1.1
+	 *
+	 * @var string
+	 */
+	public $title;
+
+	/**
+	 * Theme's status, will either be active or inactive
+	 *
+	 * @since 1.1.1
+	 *
+	 * @var string
+	 */
+	public $status;
+
+	/**
+	 * Theme's version
+	 *
+	 * @since 1.1.1
+	 *
+	 * @var string
+	 */
+	public $version;
+
+	/**
+	 * Whether there is an update available for the theme or not
+	 * will be either none or available
+	 *
+	 * @since 1.1.1
+	 *
+	 * @var string
+	 */
+	public $update;
+
+	/**
+	 * Theme's update version, will be either (undef) or the version
+	 *
+	 * @since 1.1.1
+	 *
+	 * @var string
+	 */
+	public $update_version;
+
+	/**
+	 * Theme's screenshot, an associative array of file and url
+	 *
+	 * @since 1.1.1
+	 *
+	 * @var array
+	 */
+	public $screenshot;
+
+	/**
+	 * If Auto updates have been enabled for this theme
+	 *
+	 * @since 1.1.1
+	 *
+	 * @var bool
+	 */
+	public $auto_updates_enabled;
+
+	/**
+	 * Constructor
+	 *
+	 * @since 1.1.1
+	 *
+	 * @param string   $theme_id      The theme id
+	 * @param WP_Theme $theme         object to initialize our theme
+	 * @param array    $auto_updates  array of auto update options
+	 * @param array    $theme_updates array with info about theme updates
+	 * @param WP_Theme $current_theme current theme's object
+	 */
+	public function __construct( $theme_id, $theme, $auto_updates, $theme_updates, $current_theme ) {
```

Comment:
Same as with the `Plugin` class, I think you could do everything here with just the `stylesheet` string and the rest being optional parameters.

Reply:
Made the params optional..